src/pages/AddEmployee.jsx

```diff
@@ -11,17 +11,24 @@ import {
 } from "@mui/material";
 import Grid from "@mui/material/Grid2";
 import { useMutation, useQueryClient } from "@tanstack/react-query";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { addEmployee } from "../services/employees";
 import Toast from "../components/Toast";
-function AddEmployee({ open, onClose }) {
+function AddEmployee({ open, onClose, action, employee = null }) {
   const [openToast, setOpenToast] = useState(false);
   const [toastMessage, setToastMessage] = useState("");
   const [toastSeverity, setToastSeverity] = useState("");
```

Comment:
Delvin wants all three properties to be grouped into one object since they always happen together

something like  const toastObj ={openToast:false,toastMessage:'',toastSeverity:"}
so during update you might need a spread operator if necessary